package.json

```diff
@@ -228,6 +234,7 @@
     "tippy.js": "6.3.7",
     "util": "0.12.5",
     "yup": "^0.32.11",
+    "yjs": "^13.6.18",
```

Comment:
Since a new peer dependency is necessary, please add this to the following files as well:
- [neeto-commons-frontend#package-common.json](https://github.com/bigbinary/neeto-commons-backend/blob/main/lib/neeto_commons_backend/common_files/package-common.json)

Reply:
Opened PR https://github.com/bigbinary/neeto-commons-backend/pull/3294/files and assigned it for review merging this PR